app/src/main/java/com/asma/tasky/core/presentation/components/Navigation.kt

```diff
@@ -0,0 +1,48 @@
+package com.asma.tasky.core.presentation.components
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.ScaffoldState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import com.asma.tasky.feature_management.agenda.AgendaScreen
+import com.asma.tasky.feature_authentication.presentation.login.LoginScreen
+import com.asma.tasky.feature_authentication.presentation.register.RegisterScreen
+import com.asma.tasky.core.util.Screen
+
+@ExperimentalComposeUiApi
+@Composable
+fun Navigation(
+    navController: NavHostController,
+    startDestination: String,
+    scaffoldState: ScaffoldState,
+) {
+    NavHost(
+        navController = navController,
+        startDestination = startDestination,
+        modifier = Modifier.fillMaxSize()
+    ) {
+
+        composable(Screen.LoginScreen.route) {
+            LoginScreen(
+                onNavigate = { navController.navigate(route = it) },
+                onLogin = {
+                    navController.popBackStack()
+                    navController.navigate(route = Screen.AgendaScreen.route)
+                          },
```

Comment:
Formatting seems a bit messed up here, have you considered setting up an auto-formatting tool like ktlint? 
https://www.youtube.com/watch?v=hSgPNbEcX98